tools/generate-templates.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -euo pipefail
+
+rm -rf templates || true
```

Comment:
Could it be renamed to `test/roles`?

It's a coincidence. In `ansible-lint`, the file under a directory named `templates` would be considered as a text format instead yaml format. So when `ansible-lint` runs, it would always show no Violations. 

https://github.com/ansible/ansible-lint/blob/aaab27952352edf3aa8afaffa4fc1e93ff984976/src/ansiblelint/config.py#L82

Reply:
DONE